--- admin/acinclude.m4.link	2010-07-21 12:32:40.000000000 +0000
+++ admin/acinclude.m4	2010-11-06 11:49:47.000000000 +0000
@@ -83,8 +83,8 @@
         saved_CPPFLAGS="$CPPFLAGS"
         CPPFLAGS="$CPPFLAGS -I$xerces_include_dir -I$xerces_include_dir2"
 
-        saved_LDFLAGS="$LDFLAGS"
-        LDFLAGS="$LDFLAGS $xerces_lib_flags64"
+        saved_LIBS="$LIBS"
+        LIBS="$LIBS $xerces_lib_flags64"
 
         dnl
         dnl Check Xerces headers
@@ -149,7 +149,7 @@
 	        CPPFLAGS="$CPPFLAGS -I$xerces_include_dir -I$xerces_include_dir2"
 
 	       # saved_LDFLAGS="$LDFLAGS"
-	        LDFLAGS="$LDFLAGS $xerces_lib_flags"
+	        LIBS="$LIBS $xerces_lib_flags"
 
 	        dnl
 	        dnl Check Xerces headers
@@ -212,7 +212,7 @@
 	        fi
 	
         	CPPFLAGS="$saved_CPPFLAGS"
-	        LDFLAGS="$saved_LDFLAGS"
+	        LIBS="$saved_LIBS"
 
 
 
@@ -224,7 +224,7 @@
         fi
 
         CPPFLAGS="$saved_CPPFLAGS"
-        LDFLAGS="$saved_LDFLAGS"
+        LIBS="$saved_LIBS"
 	
     fi
 
--- src/genverilog/Makefile.am.link	2010-07-21 12:32:39.000000000 +0000
+++ src/genverilog/Makefile.am	2010-11-06 11:51:14.000000000 +0000
@@ -1,7 +1,7 @@
 bin_PROGRAMS = fsmveriloggeneration
 
-INCLUDES = @QT_CFLAGS@ @XERCES_CFLAGS@
-LDADD= @QT_LIBS@ ../core/libcore.la ../xmlcreate/libxmlloadstore.la @XERCES_LDFLAGS@
+INCLUDES = @QT_CFLAGS@ @XERCES_CPPFLAGS@
+LDADD= @QT_LIBS@ ../core/libcore.la ../xmlcreate/libxmlloadstore.la @XERCES_LDFLAGS@ $(XERCES_LIBS)
 
 %.h: %.ui
 	uic -o $@ $<
--- src/simvision/Makefile.am.link	2010-07-21 12:32:02.000000000 +0000
+++ src/simvision/Makefile.am	2010-11-06 11:51:34.000000000 +0000
@@ -1,8 +1,8 @@
 noinst_LTLIBRARIES=libsim.la
 bin_PROGRAMS = createmmap createtestbench createf4tbar
 
-INCLUDES = @QT_CFLAGS@  @XERCES_CFLAGS@
-LDADD= @QT_LIBS@ ../core/libcore.la ../xmlcreate/libxmlloadstore.la @XERCES_LDFLAGS@
+INCLUDES = @QT_CFLAGS@  @XERCES_CPPFLAGS@
+LDADD= @QT_LIBS@ ../core/libcore.la ../xmlcreate/libxmlloadstore.la @XERCES_LDFLAGS@ $(XERCES_LIBS)
 
 %.h: %.ui
 	uic -o $@ $<
--- src/xmlcreate/Makefile.am.link	2010-07-21 12:32:09.000000000 +0000
+++ src/xmlcreate/Makefile.am	2010-11-06 11:49:47.000000000 +0000
@@ -1,6 +1,6 @@
 noinst_LTLIBRARIES=libxmlloadstore.la
 
-AM_CFLAGS=-fPIC @XERCES_CFLAGS@
+AM_CFLAGS=-fPIC @XERCES_CPPFLAGS@
 
 
 libxmlloadstore_la_SOURCES=xmlload.cpp  xmlload.h  xmlsave.cpp  xmlsave.h \
--- src/genvhdl/Makefile.am.link	2010-11-06 12:09:21.000000000 +0000
+++ src/genvhdl/Makefile.am	2010-11-06 12:09:31.000000000 +0000
@@ -1,7 +1,7 @@
 bin_PROGRAMS = fsmvhdlgeneration
 
 INCLUDES = @QT_CFLAGS@  $(XERCES_CFLAGS)
-LDADD= @QT_LIBS@ ../core/libcore.la ../xmlcreate/libxmlloadstore.la $(XERCES_LDFLAGS)
+LDADD= @QT_LIBS@ ../core/libcore.la ../xmlcreate/libxmlloadstore.la $(XERCES_LDFLAGS) $(XERCES_LIBS)
 
 %.h: %.ui
 	uic -o $@ $<
--- src/Makefile.am.link	2010-11-06 12:21:35.000000000 +0000
+++ src/Makefile.am	2010-11-06 12:21:52.000000000 +0000
@@ -12,4 +12,4 @@
 			corefactory.cpp corefactory.h loadfactory.h loadfactory.cpp savefactory.h savefactory.cpp XercesDefinitions.hpp
 
 
-LDADD =  gui/libgui.la simvision/libsim.la xmlcreate/libxmlloadstore.la core/libcore.la $(QT_LIBS) $(XERCES_LDFLAGS)  @PYTHON_LIB@
+LDADD =  gui/libgui.la simvision/libsim.la xmlcreate/libxmlloadstore.la core/libcore.la $(QT_LIBS) $(XERCES_LDFLAGS) $(XERCES_LIBS) @PYTHON_LIB@
